src/Resources/views/form/integrated.html.twig

```diff
@@ -87,9 +87,18 @@
             </label>
         </div>
     {% endif %}
-    <button id="paid" class="ui large primary button" type="button">
-        {{ 'payplug_sylius_payplug_plugin.ui.integrated_payment.place_order.label'|trans }}
-    </button>
+
+    <div class="payplugIntegratedPayment__container">
+        <button id="paid" class="ui large primary button" type="button">
+            {{ 'payplug_sylius_payplug_plugin.ui.integrated_payment.place_order.label'|trans }}
+        </button>
+    </div>
+    {% set config = paymentMethod.gatewayConfig.config %}
+    {% if config.deferredCapture is defined and config.deferredCapture is same as true %}
+    <div>
```

Comment:
missing indent ? maybe on purpose :shrug:

Reply:
kind of, once the if twig condition disappear in html, the div remain on same level than other. 

I can change if you prefer :+1: